perfact/zodbsync/zodbsync.py

```diff
@@ -786,7 +786,8 @@ def _playback_path(self, pathinfo):
             # Find IDs in Data.fs object not present in file system
             del_ids = [
                 a for a in srv_contents
-                if a not in contents and a != 'acl_users'
+                if a not in contents and
+                (path != '/acl_users' and a != 'acl_users')
```

Comment:
I'm wondering why line 770 seems to play no role in this…

Reply:
I am pretty sure this is crucial.
The PR currently only handles the recursive case. But when using something like `zodbsync pick`, the recursive mode is not used, all changed paths are deleted directly from the changed objects!